### PR TITLE
Add rule for hyphens similar to `yamllint`

### PR DESCRIPTION
**Pull request by dawidmalina**

### CODE REVIEW

**Comment by dawidmalina** [comment]
closes #105

File: saltlint/rules/HyphenHasSpaceRule.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import re
+
+
+class HyphenHasSpaceRule(SaltLintRule):
+    id = '219'
+    shortdesc = 'Hyphen character should be always followed by the one space.'
+    description = 'Hyphen character should be always followed by the one space.'
+    severity = 'MEDIUM'
+    tags = ['formatting']
+    version_added = 'v0.1.1'
```
Comment (jbouter) [inline saltlint/rules/HyphenHasSpaceRule.py]:
```suggestion
    version_added = 'develop'
```

File: saltlint/rules/HyphenHasSpaceRule.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Dawid Malinowski <dawidmalina@gmail.com>
+
+from saltlint.linter import SaltLintRule
```
Comment (jbouter) [inline saltlint/rules/HyphenHasSpaceRule.py]: Pylint seems to fail on the order of importing. Please move `import re` above `from saltlint.linter import SaltLintRule`

**Comment by dawidmalina** [comment]
@roaldnefs, @jbouter can you check this PR? Feel free to update or modify code if needed.

**Comment by roaldnefs** [comment]
Sorry for the late response @dawidmalina!

The last couple of weeks have been quite busy. In the upcoming period I hope to be able to continue working on salt-lint. Since your pull requests have been open for quite a while I will give them a higher priority on my action list.

**Pull request closed without merging**